substrate/frame/revive/src/lib.rs

```diff
@@ -573,6 +573,8 @@ pub mod pallet {
 		AccountUnmapped,
 		/// Tried to map an account that is already mapped.
 		AccountAlreadyMapped,
+		/// The transaction used to dry-run a contract is invalid.
+		InvalidGenericTransaction,
```

Comment:
The comment suggest this error can only occur during dry-runs, could it also explain why?

Reply:
look like this is an old error case not used anymore. :hocho:  it